latex/epoch.tex

```diff
@@ -201,696 +333,232 @@ \subsection{Stake Distribution Calculation}
-                       \sum_{\wcard\mapsto c\in\var{st}}c
-                       \right) \\
-      & ~~~~~~~pdata = \left\{
-        hk\mapsto (p, n, s)  \mathrel{\Bigg|}
-        \begin{array}{r@{\mapsto}c@{~\in~}l}
-          hk & \var{p} & \var{poolParams} \\
-          hk & \var{n} & \var{blocks} \\
-          hk & \var{s} & \var{pooledStake} \\
-        \end{array}
-      \right\} \\
-      & ~~~~~~~\var{results} = \left\{
-                 hk \mapsto \rewardOnePool{pp}{R}{n}{hk}{p}{s}{avgs}{tot}{stkeys}
-                 \mid
-                 hk\mapsto(p, n, s)\in\var{pdata} \right\} \\
-      & ~~~~~~~\var{unrealized} = \sum_{\wcard\mapsto (\wcard,~u)\in\var{results}}u \\
-      & ~~~~~~~\var{rewards} = \bigcup_{\wcard\mapsto(\var{rwds},~\wcard)}\var{rwds}
-  \end{align*}
-  \caption{The Reward Calculation}
-  \label{fig:functions:reward-calc}
-\end{figure}
-
-\clearpage
-
-\subsection{Accounting Epoch Transition}
-\label{sec:acc-trans}
-
-The Figure~\ref{fig:ts-types:accnt} gives the definitions related to epoch
-boundary ledger accounting. The figure lists the accounting fields, denoted by
-$\Accnt$, which include $\var{treasury}$ (the amount of coin currently in
-circulation, but not on the UTxO and not designated for rewards),
-$\var{reserves}$ (the amount of coin
-not yet in circulation),
-and $\var{rewardPot}$ (the total amount available for distribution of rewards).
-The figure also gives the set of necessary accounting environment
-variables, $\AccntEnv$, and the accounting state, $\AccntState$, which includes
-the delegation state and the accounting fields. The accounting state transition
-type, as all transition types in this chapter, has no signal.
-
 %%
-%% Figure - Accounting Defs
+%% Figure - Snapshots Defs
 %%
 \begin{figure}[htb]
-  \emph{Accounting Fields}
+  \emph{Snapshot environment}
   \begin{equation*}
-    \Accnt =
+    \SnapshotEnv =
     \left(
       \begin{array}{r@{~\in~}ll}
-        \var{treasury} & \Coin & \text{treasury pot}\\
-        \var{reserves} & \Coin & \text{reserve pot}\\
-        \var{rewardPot} & \Coin & \text{reward pot}\\
+        \var{slot} & \Slot & \text{current slot}\\
+        \var{pp} & \PParams & \text{protocol parameters}\\
+        \var{dstate} & \DState & \text{delegation state}\\
+        \var{pstate} & \PState & \text{pool state}\\
+        \var{blocks} & \BlocksMade & \text{blocks made}\\
       \end{array}
     \right)
   \end{equation*}
   %
-  \emph{Accounting environment}
+  \emph{Snapshots}
   \begin{equation*}
-    \AccntEnv =
+    \Snapshots =
     \left(
       \begin{array}{r@{~\in~}ll}
-        \var{slot} & \Slot & \text{last slot of epoch}\\
-        \var{pp} & \PParams & \text{protocol parameters}\\
-        \var{blocks} & \BlocksMade & \text{blocks made in the epoch}\\
+        \var{pstake_{mark}} & \PState & \text{newest stake}\\
+        \var{pstake_{set}} & \PState & \text{middle stake}\\
+        \var{pstake_{go}} & \PState & \text{oldest stake}\\
```

Comment:
Shouldn't those be of type `PooledStake` instead of `PState`?

Reply:
yes! good catch!